src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.html

```diff
@@ -1,10 +1,46 @@
-<div class="mode-select">
-  <a class="mode-select-button quickmaps-baseline-link" mat-raised-button [routerLink]="ROUTES.QUICK_MAPS_BASELINE | route"
-    [queryParams]="route.queryParams | async" routerLinkActive="active-route">Baseline</a>
-  <a class="mode-select-button quickmaps-projection-link" mat-raised-button
-    [routerLink]="((quickMapsService.micronutrientObs | async).isInImpact) ? (ROUTES.QUICK_MAPS_PROJECTION | route) : null"
-    [queryParams]="route.queryParams | async" routerLinkActive="active-route"
-    [disabled]="!(quickMapsService.micronutrientObs | async).isInImpact">Explore projections to 2050</a>
+<div class="container">
+  <div class="item-center">
+    <div *ngIf="quickMapsService.measure === MicronutrientMeasureType.DIET">
+      <h2 class="qmTitle">{{ (quickMapsService.countryObs | async)?.name }} Diet Data</h2>
+    </div>
+    <div *ngIf="quickMapsService.measure !== MicronutrientMeasureType.DIET">
```

Comment:
Because MicronutrientMeasureType is a getter (function) it might mean that it gets called on every change detection cycle.  The easy way to ensure that it isn't would be to have assigned it to a variable in the ts instead:

public readonly MEASURE_TYPE_ENUM = MicronutrientMeasureType;

Having said this, I did exactly the same thing yesterday, so I'm going to test and update my code if what I suspect is true.  If I do need to do that I can update this too if you like, only if I find it is a "problem".

Reply:
Ok good to flag it up, if its a problem that rears its head we can switcharoony